test/tests.zig

```diff
@@ -1583,6 +1592,9 @@ pub fn addModuleTests(b: *std.Build, options: ModuleTestOptions) *Step {
             run.setName(b.fmt("run test {s}", .{qualified_name}));
 
             step.dependOn(&run.step);
+        } else if (target.cpu.arch.isSpirV()) {
+            // Don't run spirv binaries
+            step.dependOn(&these_tests.step);
```

Comment:
Can you verify that this does, in fact, fully compile the tests? I suspect a `_ = these_tests.getEmittedBin()` may be necessary...

Reply:
Also, with this, do we actually need the change in `lib/compiler/test_runner.zig`?

Reply:
> Also, with this, do we actually need the change in lib/compiler/test_runner.zig?

yes there seem to a bug that makes zig always to compile `test_runner.zig` even when not running the tests